packages/beacon-node/src/execution/builder/http.ts

```diff
@@ -91,38 +102,92 @@ export class ExecutionBuilderHttp implements IExecutionBuilder {
   }
 
   async getHeader(
+    fork: ForkExecution,
     slot: Slot,
     parentHash: Root,
     proposerPubKey: BLSPubkey
   ): Promise<{
     header: allForks.ExecutionPayloadHeader;
     executionPayloadValue: Wei;
-    blobKzgCommitments?: deneb.BlobKzgCommitments;
+    blindedBlobsBundle?: deneb.BlindedBlobsBundle;
   }> {
     const res = await this.api.getHeader(slot, parentHash, proposerPubKey);
     ApiError.assert(res, "execution.builder.getheader");
     const {header, value: executionPayloadValue} = res.response.data.message;
-    const {blobKzgCommitments} = res.response.data.message as {blobKzgCommitments?: deneb.BlobKzgCommitments};
-    return {header, executionPayloadValue, blobKzgCommitments};
+    const {blindedBlobsBundle} = res.response.data.message as deneb.BuilderBid;
+    return {header, executionPayloadValue, blindedBlobsBundle};
   }
 
-  async submitBlindedBlock(signedBlock: allForks.SignedBlindedBeaconBlock): Promise<allForks.SignedBeaconBlock> {
-    const res = await this.api.submitBlindedBlock(signedBlock);
+  async submitBlindedBlock(
+    signedBlindedBlockOrContents: allForks.SignedBlindedBeaconBlockOrContents
+  ): Promise<allForks.SignedBeaconBlockOrContents> {
+    const res = await this.api.submitBlindedBlock(signedBlindedBlockOrContents);
     ApiError.assert(res, "execution.builder.submitBlindedBlock");
-    const executionPayload = res.response.data;
-    const expectedTransactionsRoot = signedBlock.message.body.executionPayloadHeader.transactionsRoot;
-    const actualTransactionsRoot = ssz.bellatrix.Transactions.hashTreeRoot(res.response.data.transactions);
+    const {data} = res.response;
+
+    let executionPayload: allForks.ExecutionPayload;
```

Comment:
will convert his from the `let ...` model to a parse return model as per the feedback from produceblockb3